pkg/fakerp/fakerp.go

```diff
@@ -145,9 +145,9 @@ func CreateOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 		return nil
 	}
 
-	err = p.CreateOrUpdate(ctx, cs, azuretemplate, oldCs != nil, deployer)
-	if err != nil {
-		return nil, err
+	perr := p.CreateOrUpdate(ctx, cs, azuretemplate, oldCs != nil, deployer)
```

Comment:
I feel like the fakerp should also return a `PluginError` here, thoughts?

Reply:
once CreateOrUpdate returns a `*PluginError`, this entire hunk can be reverted